src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java

```diff
@@ -263,6 +263,21 @@ public FFmpegOutputBuilder addOutput(URI uri) {
     return output;
   }
 
+  /**
+   * Adds new HLS(Http Live Streaming) output file.
+   * <br>
+   * ex) new FFmpegBuilder().addHlsOutput("out.m3u8")
+   *
+   * @param filename output file path
+   *
+   * @return A new {@link FFmpegHlsOutputBuilder}
+   */
+  public FFmpegHlsOutputBuilder addHlsOutput(String filename) {
+    FFmpegHlsOutputBuilder output = new FFmpegHlsOutputBuilder(this, filename);
```

Comment:
Would this now imply "-f hls", should we explicitly set the format in the FFmpegBuilder. Otherwise a caller could do:

```
FFmpegBuilder().setFormat('mp3').addHlsOutput(...)
```

which might be ambiguous.  So maybe also `checkArgument(format == null || format == "hls")`

Reply:
Oh and got the checkAgument please add a message explaining why it is expected to be unset or already set to hls.